webv2/web.go

```diff
@@ -773,8 +773,17 @@ func getSourceAndTargetProfiles(sessionState *session.SessionState, details type
 	} else {
 		sessionState.Conv.Audit.MigrationRequestId, _ = utils.GenerateName("smt-job")
 		sessionState.Conv.Audit.MigrationRequestId = strings.Replace(sessionState.Conv.Audit.MigrationRequestId, "_", "-", -1)
-		sessionState.Bucket = strings.ToLower(sessionState.Conv.Audit.MigrationRequestId)
-		sessionState.RootPath = "/"
+		if details.TargetDetails.GcsMetadataPath.GcsBucketName != "" {
+			sessionState.Bucket = details.TargetDetails.GcsMetadataPath.GcsBucketName
+			if details.TargetDetails.GcsMetadataPath.GcsBucketRootPath != "" {
+				sessionState.RootPath = details.TargetDetails.GcsMetadataPath.GcsBucketRootPath
```

Comment:
We might need to add / at the end if user's specified root path doesn't end with /

Reply:
Makes sense. Now that I think about it, technically then we should always add "/" at the end of name of gcs bucket name as well right?

Reply:
yes